aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md

```diff
@@ -84,7 +84,9 @@ As a first step you'll need to:
    Click the **Grant/revoke admin consent for {tenant}** button, and then select **Yes** when you are asked if you want to grant consent for the
    requested permissions for all account in the tenant. ![grant admin consent](docs/grant-admin-consent.png "grant admin consent")
    You need to be an Azure AD tenant admin to do this.
-   
+
+1. Then back to **Azure Active Directory**, In the left-hand navigation pane, select **Groups**, and then select `group1` which you created, choose your tenant as **Owners** and Spring Boot Sample as **Members**.
```

Comment:
`In the left-hand navigation pane` -> `in the left-hand navigation pane`.

Reply:
> and then select `group1` which you created, choose your tenant as **Owners** and Spring Boot Sample as **Members**.

Is this necessary? IMU, the only thing we need to do is set `user` as member of `group1`.